src/taco.ts

```diff
@@ -0,0 +1,55 @@
+import { DkgPublicKey, ThresholdMessageKit } from '@nucypher/nucypher-core';
+import { ethers } from 'ethers';
+
+import { DkgCoordinatorAgent } from './agents/coordinator';
+import { ThresholdDecrypter } from './characters/cbd-recipient';
+import { Enrico } from './characters/enrico';
+import { Condition, ConditionExpression } from './conditions';
+import { DkgClient } from './dkg';
+import { getPorterUri } from './porter';
+import { toBytes } from './utils';
+
+export const encrypt = async (
+  provider: ethers.providers.Provider,
+  message: string,
+  condition: Condition,
+  ritualId: number
+): Promise<ThresholdMessageKit> => {
+  const dkgRitual = await DkgClient.getFinalizedRitual(provider, ritualId);
+  return await encryptLight(message, condition, dkgRitual.dkgPublicKey);
+};
+
+export const encryptLight = async (
```

Comment:
Not sure how I feel about `Light` as the term 😅 . Can this just be an overloaded function with the same name?

Reply:
There is no support function overloading in JS/TS. I didn't have better ideas for an appropriate name, maybe we could use the `blockchainy` naming from the original TACo API issue.

Reply:
I don't have a good name for it, but I would prefer something like 

```typescript
export const encryptWithDKG = async (
``` 

than the original one